config-default.yaml

```diff
@@ -134,6 +134,11 @@ disabledTokens:
     - address: '0xFC4B8ED459e00e5400be803A9BB3954234FD50e3'
   4: [] # Rinkeby
 
+# Initial token selection for the sell token and buy token
+initialTokenSelection:
+  sellToken: DAI
+  receiveToken: USDC
```

Comment:
What about using token addresses instead?
No chance of picking the wrong token by accident, and also helps in case the token is not yet part of the UI
(Although if someone is forking it would be added anyway :thinking: )